contributing/getting-started-e2e.md

```diff
@@ -382,8 +382,8 @@ Notes:
 
 * This will force a clean build and run all the PR checks one by one.
 * Just like on Travis, a failing check will prevent subsequent checks from being run.
-* The `gulp visual-diff` check will be skipped unless you have set up a Percy account as described [here](https://github.com/ampproject/amphtml/blob/master/contributing/DEVELOPING.md#running-visual-diff-tests-locally).
-* The AMP unit and integration tests will be run on local Chrome unless you have set up a Sauce Labs account as described [here](https://github.com/ampproject/amphtml/blob/master/contributing/DEVELOPING.md#testing-on-sauce-labs).
+* The `gulp visual-diff` check will be skipped unless you have set up a Percy account as described in the [Testing](https://github.com/ampproject/amphtml/blob/master/contributing/TESTING.md#running-visual-diff-tests-locally) guide.
```

Comment:
nit: Probably best to make this a relative link to `TESTING.md` instead of a direct link to the version on `master`. That way, if someone were to browse the repository at a certain commit, they would see all the documentation at that commit.

I'd imagine there are other instances, so this could be done during a future cleanup.

Reply:
Yeah, previous links were inconsistent and was uncertain if we wanted relative linking.  I'll fix in these docs.

Reply:
Yeah, in general, when I've made incremental changes to the docs, I've copy-pasted links the `master` version without giving it much thought.

I think we could benefit from doing a global fix for absolute links across all our internal docs, and add a presubmit rule to prevent future changes that introduce absolute links. Happy to help with this. Definitely not high priority.